Polygon/include/CGAL/draw_polygon_set_2.h

```diff
@@ -0,0 +1,61 @@
+#ifndef CGAL_DRAW_POLYGON_SET_2_H
```

Comment:
Can you copy paste the license header from [here ](https://github.com/CGAL/cgal/blob/908f5811c0f1b06630716eddfcd219b5d4f1b75c/Polygon/include/CGAL/draw_polygon_with_holes_2.h#L1)

Reply:
Done.  I've kept the original author's name, as virtually none of the code is mine (I only did some reshuffling).  Also note (as you're probably already aware) that more stuff is missing (at least the documentation-related stuff is).

Reply:
I guess it also only works for the case that the edges of the polygons are straight line segments,

Reply:
@afabri The same restriction applies for polygon and polygon with holes viewers. But does CGAL polygons allow non straight line segments?

Reply:
>But does CGAL polygons allow non straight line segments?

I probably misunderstand the question, but CGAL supports polygons bounded by various weakly x-monotone curves (circles, conics, beziers, etc.).  Indeed, the current functionality only supports drawing "regular" polygons with linear edges and indeed being able to draw curved polygons would be useful, but this would require approximating the curves with polylines, which is a bit more involved.

I have working code to do that for circle-segment polygons and could try to adapt it into a viewer, if I can find some time and if you think it'd be useful.